portfolio/src/main/webapp/index.html

```diff
@@ -61,7 +89,7 @@
                 </a>
             </div>
             <p>
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras ac lacinia velit. Vivamus tristique lacinia nibh, sed tristique sem. Etiam ultrices sapien quis faucibus malesuada. Morbi hendrerit ut diam sit amet dictum. Aenean turpis nisl, fringilla convallis justo varius, blandit vehicula arcu. Sed viverra elementum odio. Nulla at eros sed massa eleifend pellentesque. Ut dignissim nisl ut viverra dictum. Curabitur id lectus viverra, feugiat libero eget, consequat tortor.
+                There are two main areas of my interest, one of them is engineering and the other is sport. In case of engineering I like solving algorithm problems and generally love programming. One of my cool projects is the reconstruction of the Stargate Atari game. It can be played on any browser but I wrote it in times when I didn’t know much about efficient programming so it is working fluently only on Chrome. My another engineering hobby is flying advanced remote controlled helicopters. To build some image of what it’s about I encourage to watch this professional <a target="_blank" href="https://www.youtube.com/watch?v=FcV8U32fSlE">flight</a> of one of the best pilots in the world. Most of my free time is spent also on playing sports. I play in local squash league, I was playing in the table tennis team at my University. Sometimes I play tennis and I love to watch ATP Tournaments. I was also playing in an ice hockey team for six years as a kid and I enjoy watching NHL matches.
```

Comment:
Wow !!

Reply:
Hah, do you really like it? :D If everything is good in your opinion I will deploy app quickly and go to week-3 starter project/